package.json

```diff
@@ -46,56 +46,57 @@
   },
   "dependencies": {
     "algoliasearch-helper": "^2.26.1",
-    "instantsearch.js": "^3.1.1"
+    "core-js": "^2.6.5",
+    "instantsearch.js": "^3.2.1"
   },
   "peerDependencies": {
-    "algoliasearch": "^3.30.0",
-    "vue": "^2.4.0"
+    "algoliasearch": "^3.32.1",
+    "vue": "^2.6.10"
   },
   "devDependencies": {
-    "@storybook/addon-knobs": "3.4.11",
-    "@storybook/addon-options": "3.4.11",
-    "@storybook/vue": "3.4.11",
-    "@vue/test-utils": "1.0.0-beta.25",
-    "algoliasearch": "3.32.0",
+    "@storybook/addon-knobs": "5.0.5",
+    "@storybook/addon-options": "5.0.5",
+    "@storybook/vue": "5.0.5",
+    "@vue/test-utils": "1.0.0-beta.29",
+    "algoliasearch": "3.32.1",
     "babel-eslint": "10.0.1",
-    "babel-jest": "23.6.0",
+    "babel-jest": "24.5.0",
     "babel-preset-es2015": "6.24.1",
     "bundlesize": "0.17.1",
     "concurrently": "4.1.0",
     "conventional-changelog-cli": "2.0.12",
-    "eslint": "5.14.1",
+    "eslint": "5.15.3",
     "eslint-config-algolia": "13.2.3",
-    "eslint-config-prettier": "3.1.0",
+    "eslint-config-prettier": "4.1.0",
     "eslint-plugin-import": "2.16.0",
-    "eslint-plugin-jest": "22.3.0",
-    "eslint-plugin-prettier": "3.0.0",
-    "eslint-plugin-vue": "4.7.1",
+    "eslint-plugin-jest": "22.4.1",
+    "eslint-plugin-prettier": "3.0.1",
+    "eslint-plugin-vue": "5.2.2",
     "instantsearch.css": "7.1.1",
-    "jest": "23.6.0",
+    "jest": "24.5.0",
     "jest-serializer-html": "6.0.0",
-    "jest-vue-preprocessor": "1.4.0",
+    "jest-vue-preprocessor": "1.5.0",
     "jest-watch-typeahead": "0.2.1",
     "lodash": "4.17.11",
-    "prettier": "1.14.3",
-    "rollup": "1.3.2",
+    "prettier": "1.16.4",
+    "rollup": "1.7.4",
     "rollup-plugin-buble": "0.19.6",
-    "rollup-plugin-commonjs": "9.2.1",
+    "rollup-plugin-commonjs": "9.2.2",
     "rollup-plugin-filesize": "6.0.1",
-    "rollup-plugin-json": "3.1.0",
+    "rollup-plugin-json": "4.0.0",
     "rollup-plugin-node-resolve": "4.0.1",
-    "rollup-plugin-replace": "2.1.0",
+    "rollup-plugin-replace": "2.1.1",
     "rollup-plugin-terser": "4.0.4",
-    "rollup-plugin-vue": "3.0.0",
+    "rollup-plugin-vue": "4.7.2",
     "rollup-watch": "4.3.1",
-    "vue": "2.5.18",
-    "vue-autosuggest": "1.8.1",
-    "vue-jest": "2.6.0",
+    "vue": "2.6.10",
+    "vue-autosuggest": "1.8.3",
+    "vue-jest": "3.0.4",
```

Comment:
Most of the major changes you did here had a negative impact on bundle size; but that could have been with a different patch version. It's fairly hard to review like this, so could you make the PR with only your changes?

Reply:
Sure. Sorry for change this file. I had to do that to be able to use my fork due the error I was receiving on build the dist files. I will revert it.